viewer/packages/data-providers/src/pointcloud-stylable-object-providers/CdfPointCloudStylableObjectProvider.ts

```diff
@@ -63,7 +63,7 @@ export class CdfPointCloudStylableObjectProvider implements PointCloudStylableOb
     return annotations;
   }
 
-  async getPointCloudObjects(modelIdentifier: ModelIdentifier): Promise<PointCloudObjectData> {
+  async getPointCloudObjects(modelIdentifier: ModelIdentifier): Promise<PointCloudObject[]> {
```

Comment:
Technically isn't a part of this PR, but if this methods fails when not getting a ModelIdentifier that is of type CdfModelIdentifier, then it should probably just accept CdfModelIdentifiers.

Reply:
I thought that would be troublesome because `getPointCloudObjects` is an inherited method taking `ModelIdentifier` as argument. But apparently Typescript allows making this a `CdfModelIdentifier`.. Seems odd to me, to be honest :thinking: